generator/tests.py

```diff
@@ -228,8 +236,22 @@ def test_render_checklist_affects_prerelease(self):
         )
         for release in ("5.1 before 5.1.8", "5.0 before 5.0.14"):
             with self.subTest(release=release):
-                expected = f"[row 1] Django\n      [row 2] {release}"
-                self.assertIn(expected, checklist_content)
+                expected = f"[row 1] Django [row 2] {release}"
+                self.assertInChecklistContent(expected, checklist_content)
+
+        cves = checklist.securityissue_set.all()
+        prenotification = [
+            "Create a new text file `prenotification-email.txt` with content",
+            "a set of security releases will be issued on Wednesday, May 7, 2025 "
+            "around 1118 UTC",
+            *(cve.headline_for_blogpost for cve in cves),
+            "Affected supported versions =========================== "
+            + " ".join(f"* Django {branch}" for branch in checklist.affected_branches),
+            *(f"* Django {version}" for version in checklist.versions),
```

Comment:
This is a little difficult to read, could this be written out for the test?

Reply:
Good point, tests should be easy to read and not require too much brain cycles, I will expand that.